<ui version="4.0" >
 <class>dlgGetRunInfo</class>
 <widget class="QDialog" name="dlgGetRunInfo" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>563</width>
    <height>243</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="MinimumExpanding" hsizetype="Preferred" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle" >
   <string>Details for simulation run</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7" >
     <property name="margin" >
      <number>0</number>
     </property>
     <item>
      <widget class="QFrame" name="frame" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Preferred" hsizetype="MinimumExpanding" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize" >
        <size>
         <width>450</width>
         <height>0</height>
        </size>
       </property>
       <property name="frameShadow" >
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QGridLayout" name="gridLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <item row="0" column="0" >
         <widget class="QLabel" name="label_5" >
          <property name="text" >
           <string>Run id:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1" >
         <widget class="QLabel" name="lblRunId" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Preferred" hsizetype="MinimumExpanding" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize" >
           <size>
            <width>250</width>
            <height>25</height>
           </size>
          </property>
          <property name="autoFillBackground" >
           <bool>false</bool>
          </property>
          <property name="frameShape" >
           <enum>QFrame::Panel</enum>
          </property>
          <property name="frameShadow" >
           <enum>QFrame::Sunken</enum>
          </property>
          <property name="text" >
           <string>...</string>
          </property>
          <property name="textFormat" >
           <enum>Qt::PlainText</enum>
          </property>
          <property name="textInteractionFlags" >
           <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
          </property>
         </widget>
        </item>
        <item row="1" column="0" >
         <widget class="QLabel" name="label_4" >
          <property name="text" >
           <string>Scenario:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1" >
         <widget class="QLabel" name="lblScenario_name" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize" >
           <size>
            <width>0</width>
            <height>25</height>
           </size>
          </property>
          <property name="autoFillBackground" >
           <bool>false</bool>
          </property>
          <property name="frameShape" >
           <enum>QFrame::Panel</enum>
          </property>
          <property name="frameShadow" >
           <enum>QFrame::Sunken</enum>
          </property>
          <property name="text" >
           <string>...</string>
          </property>
          <property name="textFormat" >
           <enum>Qt::PlainText</enum>
          </property>
          <property name="textInteractionFlags" >
           <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
          </property>
         </widget>
        </item>
        <item row="2" column="0" >
         <widget class="QLabel" name="label" >
          <property name="text" >
           <string>Run name:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1" >
         <widget class="QLabel" name="lblRun_name" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize" >
           <size>
            <width>0</width>
            <height>25</height>
           </size>
          </property>
          <property name="autoFillBackground" >
           <bool>false</bool>
          </property>
          <property name="frameShape" >
           <enum>QFrame::Panel</enum>
          </property>
          <property name="frameShadow" >
           <enum>QFrame::Sunken</enum>
          </property>
          <property name="text" >
           <string>...</string>
          </property>
          <property name="textFormat" >
           <enum>Qt::PlainText</enum>
          </property>
          <property name="wordWrap" >
           <bool>true</bool>
          </property>
          <property name="textInteractionFlags" >
           <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
          </property>
         </widget>
        </item>
        <item row="3" column="0" >
         <widget class="QLabel" name="label_2" >
          <property name="text" >
           <string>Years run:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1" >
         <widget class="QLabel" name="lblYears_run" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize" >
           <size>
            <width>0</width>
            <height>25</height>
           </size>
          </property>
          <property name="autoFillBackground" >
           <bool>false</bool>
          </property>
          <property name="frameShape" >
           <enum>QFrame::Panel</enum>
          </property>
          <property name="frameShadow" >
           <enum>QFrame::Sunken</enum>
          </property>
          <property name="text" >
           <string>...</string>
          </property>
          <property name="textFormat" >
           <enum>Qt::PlainText</enum>
          </property>
          <property name="textInteractionFlags" >
           <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
          </property>
         </widget>
        </item>
        <item row="4" column="0" >
         <widget class="QLabel" name="label_3" >
          <property name="text" >
           <string>Cache directory:</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0" >
         <spacer name="verticalSpacer" >
          <property name="orientation" >
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>20</width>
            <height>0</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="4" column="1" >
         <layout class="QHBoxLayout" name="horizontalLayout_5" >
          <item>
           <widget class="QLabel" name="lblCache_directory" >
            <property name="minimumSize" >
             <size>
              <width>0</width>
              <height>25</height>
             </size>
            </property>
            <property name="autoFillBackground" >
             <bool>false</bool>
            </property>
            <property name="frameShape" >
             <enum>QFrame::Panel</enum>
            </property>
            <property name="frameShadow" >
             <enum>QFrame::Sunken</enum>
            </property>
            <property name="text" >
             <string>...</string>
            </property>
            <property name="wordWrap" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="tb_select_cachedir" >
            <property name="minimumSize" >
             <size>
              <width>25</width>
              <height>25</height>
             </size>
            </property>
            <property name="text" >
             <string>...</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
